lib/components/badge/badge.dart

```diff
@@ -40,20 +39,20 @@ enum TUIBadgeSize {
     }
   }
 
-  TextStyle _getStyle() {
+  TextStyle _getStyle(TUIThemeData theme) {
     switch (this) {
```

Comment:
We can extract `theme.colors` to `final colors = theme.colors;` as we have in other places.